product_customer_code_sale/README.rst

```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Product Customer Code For sale
+==============================
+Based on product_supplierinfo_for_customer, this module loads in every sale order the
+customer code defined in the product.
+
```

Comment:
I think the Usage section with the "Try me on runbot" button is a must:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst